package.json

```
@@ -55,8 +55,8 @@
     "typeorm": "^0.2.24"
   },
   "peerDependencies": {
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1"
+    "react": "^16.13.1 || 17",
+    "react-dom": "^16.13.1 || 17"
```

May I suggest using this instead?

```suggestion
    "react": ">=16.13.1",
    "react-dom": ">=16.13.1"
```

---

Thanks both! Am thinking we tweak that further to `"^16.13.1 || ^17"`?

Will try and get this released over the weekend.

---

It's hard to tell without knowing how breaking an eventual React 18 would be but in the past upgrading has been very smooth as long as deprecations are fixed in a timely manner.

Also, npm seems to be somewhat nudging toward having loose peer dependencies to avoid, for example, installing two copies of React when a user tries out a v18 pre-release.

That's why I personally prefer the `>=` approach but I don't have a very strong opinion either way!

---

I don't have much of an opinion either, but tweaked it to `^16.13.1 || ^17` anyway

---

Thanks!